docs/contributors/contributors-guide.md

```diff
@@ -0,0 +1,178 @@
+
+## Insights: Connecting Your Repositories
+<!-- revise this one tomorrow -->
+OpenSauced Insights provides contributors with a visual representation of your favorite project's health and contributor engagement. This feature also offers real-time data and analytics to understand contribution patterns, identify key contributors, and measure the project's growth. Insights can help you assess team activity, understand when bottlenecks occur, identify where resource allocation is needed, plan and identify key contributors, understand alumni contributors, or spot emerging trends.
+
+Insights features allow you to:
+
+- track the contributions of a specific team, a specific repository, or a group of repositories,
+- view the number of pull requests that are open, closed, in draft, and merged,
+- view the average number of days it takes for a pull request to be merged (PR velocity),
+- identify the top contributors to your project,
+- identify the current activity level and trends in activity of your contributors.
+
+### Creating a New Insight Page
+<!-- work on this tomorrow -->
+To connect your repositories, click the "Insights" tab on the dashboard. You will be redirected to a page where you can create your new Insight Page.
+
+There are two ways to add repositories to your Insight Page:
+
+1. **Sync GitHub organization**. Syncing your GitHub organization is a good idea for someone who seeks a comprehensive, unified view of your organization's open source project activities and trends, enabling streamlined management, strategic decision-making, and efficient allocation of resources across multiple repositories.
+2. **Connect individual repositories**. Connecting individual repositories is a good idea for someone who wants to track the contributions of a specific repository or group of repositories. For example, you might want to track projects you maintain that are not under the same organization, or you may want to track projects that align with your team's objectives.
+
+![sync team gif](../../static/gif/team-sync-insights.gif)
+
+### Using Your Insights Page
+
+Once you have connected your repositories, you will be redirected to your Insight Page. Here, you will see a dashboard with an overview of the repositories and the contributors who have contributed to them.
+
+There are three tabs that provide you with more information on these repositories:
+
+#### Reports
+
+The Reports tab allows paid users to filter for more information about their repositories over the last 30 days and to generate a Downloadable CSV. Filters include: Top 100 Repos, Minimum 5 Contributors, Recent, Most Active, and Spammed.
+
+#### Repositories
+
+The Repositories tab allows you to view more detailed information on each repository, including Activity, PR Overview, PR Velocity, Spam, Contributors, and activity over the last 30 days. To learn more about these features, see [Understanding Insights Data](understanding-insights.md).
```

Comment:
The links for understanding insights and lists isn't right. We can link to what's in the maintainers guide for now, but we should write an issue for creating specific pages for contributors to make it tailored to their needs.